docs/source/installation.rst

```diff
@@ -375,27 +375,62 @@ complete, as it downloads and installs a number of dependencies.
 Database setup
 --------------
 
-:program:`Yith Library Server` uses `MongoDB <http://www.mongodb.org>`_ for
-its storage needs. In modern operating systems it is pretty easy to install
-MongoDB. As an example, on an Ubuntu Linux system you can use the following
+:program:`Yith Library Server` uses `PostgreSQL
+<http://www.postgresql.org>`_ for its storage needs. In modern
+operating systems it is pretty easy to install Postgres. As an
+example, on a Debian or Ubuntu Linux system you can use the following
 commands
 
 .. code-block:: text
 
-   $ sudo apt-get install mongodb mongodb-server
-   $ sudo service mongodb start
+   $ sudo apt-get install postgresql postgresql-contrib
 
 Or, in a Fedora Linux system, the equivalent commands are:
 
 .. code-block:: text
 
-   $ sudo yum install mongodb mongodb-server
-   $ sudo systemctl start mongod.service
+   $ sudo yum install postgresql-server postgresql-contrib
+   $ sudo systemctl enable postgresql
 
-By default :program:`Yith Library Server` will look for a MongoDB server in
-``localhost``, listening on port ``27017`` and will use a database
-named ``yith-library``. Of couse you can change these settings as explained
-in the :ref:`configuration_chapter` section.
+Once PostgreSQL is installed you will need to configure it. By default
+:program:`Yith Library Server` will connect to a ``yithlibrary``
+database in ``localhost`` on port ``5432`` (PostgreSQL's default).
+
+You will need to create a ``yithlibrary`` user in PostgreSQL and the
+``yithlibrary`` database. Managing and configuring a PostgreSQL server
+goes beyond the scope of this document, but you can execute these
+commands to create the user and database:
+
+.. code-block:: text
+
+   $ sudo su - postgres
+   $ createuser -P yithlibrary
+   <input user password that you want to use, keep it secret!>
+   $ createdb -E UTF8 -O yithlibrary yithlibrary
+
+If ``createdb`` complains about the UTF8 encoding it might be
+because your PostgreSQL cluster is using a different incompatible
+locale (frequently SQL_ASCII). If your server is a fresh install it
+would be advisable to `recreate the cluster with a proper locale
+<https://wiki.debian.org/PostgreSql#Changing_Debian_default_installation>`_.
+
+Once the user and database are created put the password for the
+``yithlibrary`` user in the ``database_url`` setting as explained in
+the :ref:`configuration_chapter` section.
+
+The last step is to create the tables that :program:`Yith Library
+Server` needs in the database. To do this just execute this command
+inside the `env` virtualenv:
+
+.. code-block:: text
+
+   $ bin/yith_create_db
+
+Useful links:
+
+* PostgreSQL on Debian: https://wiki.debian.org/PostgreSql
+* PostgreSQL on Fedora: https://fedoraproject.org/wiki/PostgreSQL
+* PostgreSQL documentation: http://www.postgresql.org/docs/current/
```

Comment:
Can you also mention that you need to install an extension for the uuid creation function and also the configuration setting for the max allowed transactions?